File: Morning Snack/Python/count_leap_year.py
divisorOne = 4;
divisorTwo = 100;
divisorThree = 400;
count = 0;
for year in range (1900, 2025):
    if year % divisorOne == 0 and year % divisorTwo != 0 or year % divisorThree == 0:
        count +=1
        print(count)

File: Morning Snack/JavaScript/countleapyear.js
let divisorOne = 4;
let divisorTwo = 100;
let divisorThree = 400;
let year;
let count = 0;
for (year = 1900; year <= 2025; year++){
    if(year % divisorOne === 0 && year % divisorTwo != 0 || year % divisorThree === 0){
            count +=1
                 console.log(count);
        }
    }
